src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -91,6 +91,14 @@ int MacroAssembler::align(int modulus, int extra_offset) {
   return (int)(offset() - before);
 }
 
+void MacroAssembler::assert_alignment(int alignment) {
+  assert_alignment(pc(), alignment);
+}
+
+void MacroAssembler::assert_alignment(address pc, int alignment) {
+  assert(is_aligned(pc, alignment), "bad alignment");
+}
+
```

Comment:
Would you mind a few more tweaks?
I don't think we need two versions here. Maybe we can only keep the second one and remove the first one here?
Then the call sites will look more consistent passing an explict 'pc' argument.
Also we should move the function definition to macroAssembler_riscv.hpp.

Reply:
Of course, revised and add `static inline void` for it in the hpp file.